External/Plugins/ASCompletion/Model/ASFileParser.cs

```diff
@@ -1189,7 +1189,7 @@ public void ParseSrc(FileModel fileModel, string ba, bool allowBaReExtract)
                     else
                     {
                         // valid chars for identifiers
-                        if (c1 >= 'A' && c1 <= 'Z')
+                        if (char.IsLetter(c1))
```

Comment:
Accented characters in identifiers? Is it worth the additional processing effort?